secretservice/secretservice_linux_test.go

```diff
@@ -7,8 +7,14 @@ import (
 	"github.com/docker/docker-credential-helpers/credentials"
 )
 
+// These tests are not runnable from a headless environment such as
+// Docker or a CI pipeline due to the DBus "prompt" interface being called
+// when creating and unlocking a keychain.
```

Comment:
Need to find a headless CLI tool to be able to run `secretservice` tests suite or we could mock prompter that connects to DBus.